### ISSUE TITLE
Replacing clt+R by ctl+Y

### ISSUE THREAD

**Issue by png2** [q55x8x/Peronality-Creator#19]
Isn't clt+Y the usual shortcut for undo? I never saw clt+R anywhere... 

Especially since ctl+R is the replace feature in a lot of editors that could be dangerous

**Comment by q55x8x** [comment]
Yes this really needs some thinking or maybe a survey on the community to get right.
As for other keyboard layouts this is always different. eg.. for me undoing is normally ctrl-z and ctrl-r is redo and ctrl-shift-f is replace...
